themes/codefor-theme/layouts/partials/footer.html

```diff
@@ -2,9 +2,10 @@
     <div class="content">
         <div>
             <ul class="list-unstyled">
+                <li><a class="text-dark no-underline" href="/grundsaetze">Code of Conduct</a></li>
+                <li><a class="text-dark no-underline" href="/presse">Presse</a></li>
```

Comment:
Würde "Presse" zu "Presseschau" ändern, Link heißt zwar Presse, aber die Seite Presseschau

Reply:
Renamed in 8c4aaeac90b6adab842a91cda9237277dbc35df0